llvm/Makefile.am

```diff
@@ -25,6 +29,36 @@ bump-branch-llvm: __bump-branch-llvm
 bump-current-llvm: __bump-current-version-llvm
 
 clean-local:
-	$(RM) -r $(LLVM_PATH)
+	$(RM) -r $(LLVM_PATH) $(LLVM_BUILD) $(LLVM_PREFIX)
 
 EXTRA_DIST=SUBMODULES.json
+
+NINJA:=$(shell which ninja)
+
+$(LLVM_BUILD) $(LLVM_PREFIX):
+	mkdir -p $@
+
+.PHONY: all-local
+all-local: configure-llvm build-llvm install-llvm
+
+$(LLVM_PATH)/CMakeLists.txt: | reset-llvm
+
+.PHONY: configure-llvm
+configure-llvm: $(LLVM_PATH)/CMakeLists.txt | $(LLVM_BUILD)
+	cd $(LLVM_BUILD) && cmake \
+		$(if $(NINJA),-G Ninja) \
+		-DCMAKE_INSTALL_PREFIX="$(LLVM_PREFIX)" \
+		-DCMAKE_BUILD_TYPE=Release \
```

Comment:
Can we  enable asserts optionally? We trigger some currently..

Reply:
Addressed

Reply:
We should only do this optionally they slow down llvm a lot.

Reply:
@vargaz what do you mean "optionally?" Do we have a flag anywhere? How do you want to set this? Will we disable it for release? Should I leave it enabled or disabled for this PR?

Reply:
It should not be on by default.

Reply:
It's a configure option now